ui/src/components/data-entities/customTooltip.js

```diff
@@ -17,19 +18,20 @@ export default class CustomTooltip extends Component {
 
       if (data.length > 0) {
         return (
-              <div className={"custom-tooltip"} >
+              <div className={'custom-tooltip'} >
               <span>
-                <div className={"body"}>
-                  {data.split(",").map( item => <p key={ _.uniqueId('tooltipItem-')} >{item}</p>)}
+                <div className={'body'}>
+                  {data.split(',').map( item => <p key={ _.uniqueId('tooltipItem-')} >{item}</p>)}
                 </div>
               </span>
             </div>
         );
       }
     }
-
-    return (<div className={"custom-tooltip ag-tooltip-hiding"} ></div>)
-
-
+    return (<div className={'custom-tooltip ag-tooltip-hiding'} ></div>);
   }
-}
\ No newline at end of file
+}
+
```

Comment:
Whats this for?

Reply:
It would be good to know all the possibilities of  "value".

I left "any" because I saw the typedef which means it might comes as something else than 'object'

Reply:
Yes you did start to explain